### PR TITLE
avoid importing files twice, fixes "Duplicate name in namespace" bug

### PR DESCRIPTION
**Pull request by b-ono**
This fixes a bug where the same file got imported multiple times. 

For example if there is a file A,B and C. B imports A and C imports B and A. This caused a "Duplicate name in namespace ..." exception.

### CODE REVIEW

**Comment by cbrand** [comment]
Hey thanks for the pull request!

I wrote a unit test for the issue and yes, that's in there. However, the merge request you provided is also failing the unit test (and other tests). As far as I can see you renamed the "nodePath" to "path" in the parser/protobuf.js file.

This is overwritten by the local "path" variable which is introduced in line 39. You can use the unit test I wrote in the b-ono-master branch. Can you please fix it? If that is done and the jslint issues are addressed which you can see when executing "gulp js:lint" I gladly add it to the master branch.

File: parser/protobuf.js [hunk 1]
```diff
@@ -24,30 +27,25 @@ parseProtobufString = function (protoBufString, options, loaded) {
...
-            }
             // Merge imports, try include paths
-            var fileFound = false;
-            var path = options.path || [];
-            for (var j = 0; j < path.length; ++j) {
-                var importFilename = nodePath.resolve(path[j] + '/', imports[i]);
-                if (!fs.existsSync(importFilename)) {
-                    continue;
+            (function() {
+                var path = options.path || [];
```
Comment (cbrand) [inline parser/protobuf.js]: The path viarable here is overwrites the outer scope "path" variable which imports the node path module.

**Comment by b-ono** [comment]
Should be fine now :smile: sry for the supid errors

**Comment by cbrand** [comment]
Looks good! :+1: Thank you for the contribution and no worries.

**Comment by cbrand** [comment]
Available on npm in version [1.2.2](https://www.npmjs.com/package/gulp-protobufjs/)

**Pull request merged**